packages/nvidia_nat_langchain/src/nat/plugins/langchain/langsmith/langsmith_optimization_callback.py

```
@@ -0,0 +1,372 @@
+                break
+        base = base.lower().replace(".", "-").replace("_", "-")
+
+        pattern = re.compile(re.escape(base) + r"-run-(\d+)$")
+        max_run = 0
+        try:
+            for prompt in self._client.list_prompts(query=base).repos:
+                match = pattern.match(prompt.repo_handle)
+                if match:
+                    max_run = max(max_run, int(match.group(1)))
+        except Exception:
+            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)